spec/lib/sm/api/message_drafts_spec.rb

```diff
@@ -5,7 +5,7 @@
 describe 'sm client' do
   describe 'message drafts' do
     before(:all) do
-      VCR.use_cassette 'sm_client/session', record: :new_episodes do
+      VCR.use_cassette 'sm_client/message_drafts_session', record: :new_episodes do
```

Comment:
I changed each .spec file to use a separate session cassette, as it made it easier to repeatedly re-run/re-generate the cassettes. Otherwise again, it would wind up passing obfuscated session token value to MHV and erroring out.